packages/@webex/plugin-meetings/src/member/member.types.ts

```diff
@@ -0,0 +1,13 @@
+// values are inherited from locus so don't update these
+export enum MediaStatus {
```

Comment:
the same enum and also same IMediaStatus interface are already defined in packages/@webex/plugin-meetings/src/member/types.ts file

Reply:
Removed this file.
PR in next for this changes - https://github.com/webex/webex-js-sdk/pull/3513